src/main/java/com/epam/auto/test/level2/classes/ships/RiverPort.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.auto.test.level2.classes.ships;
+
+import java.util.List;
+
+public class RiverPort {
+    private String name;
+    private List<Berth> berthList;
+
+    public RiverPort(String name, List<Berth> berthList) {
+        this.name = name;
+        this.berthList = berthList;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
For what and where do you use Port name? Please follow YAGNI principle https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it